config/usr/share/plymouth/plymouthd.defaults

```diff
@@ -0,0 +1,3 @@
+[Daemon]
+Theme=vendor
+ThemeDir=/run/mnt/data/plymouth/themes
```

Comment:
Right now /run/mnt/data/ contains "system-data", "user-data" which are mapped to "/" and "/home" respectively. Is the thinking behind this that there should be a third hierarchy or could this be integrated into the system-data/ subtree? If it gets integrated some "writable-path" changes may be needed. But maybe I'm missing the bigger pictures, how will the data get into that dir in the first place? Is it via gadget.yaml?

Reply:
Yes, the plan is to be able to customize the theme by including the content from the gadget. I'm putting this in the data partition (encrypted) to avoid possible security issues as we discussed previously. Looking at this now, it is true that having a `plymouth` folder at the same level as `system-data` and `user-data` is a bit strange... but not sure if integrating it in `system-data` would make sense. Maybe a new folder like `gadget-data` would make sense?

Reply:
Thanks for elaborating this. I like the new design to put it there but agree that maybe "plymouth" is a bit too specific. I like `gadget-data` - maybe @niemeyer or @pedronis have an opinion about the name? In general we are very careful about (user visible) naming so I would love to get the input of one of them here.